build/images/ovs/build.sh

```diff
@@ -23,11 +23,63 @@ function echoerr {
     >&2 echo "$@"
 }
 
+_usage="Usage: OVS_VERSION=<VERSION> $0 [--pull] [--push] [--platform <PLATFORM>]
+Build the antrea/ovs:<VERSION> image.
```

Comment:
Question: should we add platform to the image alias?

Reply:
I do have a long term plan to move to multi-platform Docker manifests for all images (openvswitch, base, antrea), but not in the near future (I'm waiting for Github to support self-hosted runners for public repos). If we add platform to the image name, it will make the build chain a bit harder: when building the `base` image for arm, it will need to build from the correct `openvswitch` image (e.g. `antrea/openvswitch-arm:<VERSION>`). At the moment, I always build the base images from scratch, for every arm build, leveraging a local build cache when possible. I have chosen not to push them to the Docker registry until I make the move to multi-platform Docker manifests. So the only base images we have in the registry are the amd64 ones.

Reply:
Ok. Thanks for the explanation!